GameData/ROEngines/PartConfigs/RL10C_SSTU.cfg

```diff
@@ -0,0 +1,140 @@
+	description = Hydrolox restartable expander-cycle vacuum engine. Initially developed by Pratt & Whitney designated as XLR115 and with a vacuum specific impulse of 422 seconds and rated thrust of only 66 kN. Later, the management transferred to the Marshall Space Flight Center and the engine was renamed into RL10. Since then, it has been upgraded many times, now achieving a specific impulse of up to 465 seconds and over 100 kN of thrust. Used by the workhorse Atlas Centaur high energy upper stage, the Saturn I upper stage and the Delta IV Cryogenic Second Stage (DCSS). Includes the B and CECE variants.
+	
+	tags = rl10, rl10c, rl10c-1, rl-10, hydrolox, dcss, cece
+	
+	engineType = RL10
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+    childStageOffset = 1
+	
+	TechRequired = hydrolox2009
+	cost = 500
+	entryCost = 0
+	
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = RL10B-2-ThrustTransform
+		engineID = RL10B2
+	}
+	
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = RL10B-2-Bell
+		gimbalRange = 4
+		useGimbalResponseSpeed = true
+		gimbalResponseSpeed = 16
+	}
+	
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 1
+		fxMax = 1
+		maxDistance = 120
+		falloff = 2
+		thrustTransformName = SmokeTransform
+	}
+	
+
+	MODULE
+	{
+		name = ROEModelConstraint
+		LOOK_CONST
+		{
+			transformName = RL10B-2-Strut-LeftUpper
+			targetName = RL10B-2-Target-LeftLower
+		}
+		LOOK_CONST
+		{
+			transformName = RL10B-2-Strut-LeftLower
+			targetName = RL10B-2-Target-LeftUpper
+		}
+		LOOK_CONST
+		{
+			transformName = RL10B-2-Strut-RightUpper
+			targetName = RL10B-2-Target-RightLower
+		}
+		LOOK_CONST
+		{
+			transformName = RL10B-2-Strut-RightLower
+			targetName = RL10B-2-Target-RightUpper
+		}
+		LOOK_CONST
+		{
+			transformName = RL10B-2-FuelJointLeft
+			targetName = RL10B-2-Target-FuelLeft
+		}
+		LOOK_CONST
+		{
+			transformName = RL10B-2-FuelJointRight
+			targetName = RL10B-2-Target-FuelRight
+		}
+	}
+	
+}
+
+@PART[ROE-RL10C]:AFTER[RealismOverhaulEngines]
+{
+	@title = RL10-C Engine
+	@manufacturer = Aerojet Rocketdyne
+	@description = Hydrolox restartable expander-cycle vacuum engine. Initially developed by Pratt & Whitney designated as XLR115 and with a vacuum specific impulse of 422 seconds and rated thrust of only 66 kN. Later, the management transferred to the Marshall Space Flight Center and the engine was renamed into RL10. Since then, it has been upgraded many times, now achieving a specific impulse of up to 465 seconds and over 100 kN of thrust. Used by the workhorse Atlas Centaur high energy upper stage, the Saturn I upper stage and the Delta IV Cryogenic Second Stage (DCSS).
+
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@configuration = RL10B-2
+		!CONFIG[RL10A-4] {}
+		!CONFIG[RL10A-4-1-2] {}
+		!CONFIG[RL10A-1] {}
+		!CONFIG[RL10A-3-1] {}		
+		!CONFIG[RL10A-3-3] {}
+		!CONFIG[RL10A-3-3A] {}
+		!CONFIG[RL10A-5] {}
+		!CONFIG[RL10B-2] {}
+		!CONFIG[RL10C-2-1] {}
```

Comment:
These two should be removed?

Reply:
The RL10C-2-1 and C-3 use the B-2 nozzle, not the cut down C nozzle. They are added to the RL10B